src/PhappCommandBase.php

```diff
@@ -100,7 +109,12 @@ protected function getPhappEnviromentVariables($directory = './') {
     }
 
     $env_vars = [];
+    // Add env vars from manifest.
+    if ($manifest) {
```

Comment:
this misses a fallback to $this->manifest when the method is called for regular execution on the root level